client/state/login/test/reducer.js

```diff
@@ -28,103 +33,143 @@ describe( 'reducer', () => {
 			'requestError',
 			'requestSuccess',
 			'twoFactorAuth',
+			'isRequestingTwoFactorAuth',
+			'twoFactorAuthRequestError',
 		] );
 	} );
 
-	describe( 'isRequesting', () => {
-		it( 'should default to a false', () => {
-			const state = isRequesting( undefined, {} );
+	const isRequestingReducers = [
+		{
+			name: 'isRequesting',
+			func: isRequesting,
+			loadingAction: LOGIN_REQUEST,
+			failureAction: LOGIN_REQUEST_FAILURE,
+			successAction: LOGIN_REQUEST_SUCCESS,
+		},
+		{
+			name: 'isRequestingTwoFactorAuth',
+			func: isRequestingTwoFactorAuth,
+			loadingAction: TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST,
+			failureAction: TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST_FAILURE,
+			successAction: TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST_SUCCESS,
+		},
+	];
+
+	isRequestingReducers.forEach( currentReducer => {
+		describe( `${ currentReducer.name }`, () => {
```

Comment:
Really not a fan of generating testcases this way. We should keep tests simple and not be afraid of duplication.